tabbycat/adjfeedback/views.py

```diff
@@ -373,7 +373,7 @@ def get_tables(self):
                 'tooltip': _("Institution"),
             }, [team.institution.code if team.institution else TabbycatTableBuilder.BLANK_TEXT for team in tournament.team_set.all()])
 
-        adjudicators = tournament.adjudicator_set.all()
```

Comment:
What's the use of this line? (Why keep it at all?)

Reply:
I don't know.

Reply:
I suggest you remove the line completely; the statement does not change the state, so does nothing useful by itself (not assigned).